src/kratos.ts

```diff
@@ -74,6 +74,16 @@ export const clearAuthentication = async (
   setCookie(req, res, 'auth', undefined);
   setCookie(req, res, 'kratosContinuity', undefined);
   setCookie(req, res, 'kratos', undefined);
+
+  if (req.meter) {
+    req.meter
+      .createCounter('clear_authentication', {
```

Comment:
Ideally we should store a reference to this metric somewhere and then refer to it, don't we?

Reply:
Do you mean the counter itself?